apps/src/whiteboard/entrypoint.ts

```diff
@@ -0,0 +1,14 @@
+import {lazy} from 'react';
+
+import {Lab2EntryPoint} from '@cdo/apps/lab2/types';
+
+export const WhiteboardEntryPoint: Lab2EntryPoint = {
+  themes: ['Light'],
```

Comment:
Can we support dark mode immediately?  Partly to make sure we don't get stuck without a good solution later?  And would this have any impact on the name `Whiteboard`?

Reply:
What would "support" entail? I can update this array, but not sure it does anything at the moment 😅 

Excalidraw has light/dark by default, and supports passing in your own colors:
https://docs.excalidraw.com/docs/@excalidraw/excalidraw/customizing-styles

Here's the default dark mode:

<img width="1680" height="1003" alt="image" src="https://github.com/user-attachments/assets/dba373d9-d441-4088-bb3a-97828933490e" />

Reply:
it becomes "Chalkboard" in dark mode 😂

Reply:
Oh, glad to see the library supports it.  I think support would entail having both entries in that array, and then making sure it renders in light or dark mode like in your screenshot depending on the current mode.  Maybe it's fine to do as a fast follow. 

Only other name ideas are `Sketch` or `Board`, but if `Whiteboard` is settled then no objections.

Reply:
> it becomes "Chalkboard" in dark mode 😂

My pitch a couple months ago was that we render the lines as chalk in dark mode.  I think it would be quite fun.

Reply:
Heads up @breville I checked in with Dani and we're rolling with Sketch Lab, thanks for the suggestion!